### DISCUSSION TITLE
Get pdf bytes from blob after filling out PDF form fields (editable PDF)

### DISCUSSION THREAD

**Discussion by sandeep2244** [Hopding/pdf-lib#1311] [Q&A]
Is it possible with this library to allow the user to edit form fields in a PDF, then instead of downloading the modified file, export it as a blob so that it can be uploaded to a server.

Implemented solution as below. 

Load PDF in iframe as blob URL.  (done)
Edit PDF (done)
Get Edited PDF with value in bytes array to sent it to server. 

I tried to get it from blob but its not updated PDF. 

Appreciate any help.
Thanks

**Comment by AYGTX** [comment]
It is possible
start here 
https://medium.com/factory-mind/angular-pdf-forms-fa72b15c3fbd
